Sources/UIKitNavigation/Navigation/NavigationStackController.swift

```diff
@@ -191,7 +191,8 @@
 
       override func responds(to aSelector: Selector!) -> Bool {
         aSelector == #selector(navigationController(_:didShow:animated:))
-          || base?.responds(to: aSelector) ?? false
+          || MainActor.assumeIsolated { base?.responds(to: aSelector) }
```

Comment:
Open to better ideas here. Technically isolation is not guaranteed here. If `responds(to:)` is invoked on a background thread, this line will crash. In practice UIKit doesn't seem to call this on a non-background thread, but if we can avoid even the remote possibility of crashing here, we probably should.